src/sentry/event_manager.py

```diff
@@ -875,6 +883,19 @@ def save(self, project_id, raw=False, assume_normalized=False):
 
         return event
 
+    def _get_event(self, project_id, event_id):
+        if options.get("store.use-nodestore"):
+            node_data = nodestore.get(Event.generate_node_id(project_id, event_id))
+            if node_data:
+                return eventstore.get_event_by_id(project_id, event_id)
```

Comment:
Why do we need to load the event object from snuba when we already have the event content from nodestore ?

Reply:
Good point, we should definitely avoid this